### ISSUE TITLE
Restore backup

### ISSUE THREAD

**Issue by rustincole** [AndreAle94/moneywallet#250]
Hi,

I exported my moneywallet as a .csv and .xls. Is it possible to restore this when installing the app on a new phone?
Thanks in advance

**Comment by EdMaxPrime** [comment]
Yes, there are instructions for all formats here [https://edmaxprime.github.io/moneywallet/advanced/import-export/](https://edmaxprime.github.io/moneywallet/advanced/import-export/)

[Restoring from CSV](https://edmaxprime.github.io/moneywallet/advanced/csv/#csv-import) includes transactions, wallets, people, places, and categories. However, only a JSON backup will include events, debts, recurrences, models, etc.
